e2e/ctl_v3_make_mirror_test.go

```diff
@@ -123,3 +125,54 @@ func makeMirrorModifyPrefixTest(cx ctlCtx) {
 		cx.t.Fatal(err)
 	}
 }
+
+func makeMirrorNoDestPrefixTest(cx ctlCtx) {
+	// set up another cluster to mirror with
+	mirrorcfg := configAutoTLS
+	mirrorcfg.clusterSize = 1
+	mirrorcfg.basePort = 10000
+	mirrorctx := ctlCtx{
+		t:           cx.t,
+		cfg:         mirrorcfg,
+		dialTimeout: 7 * time.Second,
+	}
+
+	mirrorepc, err := newEtcdProcessCluster(&mirrorctx.cfg)
+	if err != nil {
+		cx.t.Fatalf("could not start etcd process cluster (%v)", err)
+	}
+	mirrorctx.epc = mirrorepc
+
+	defer func() {
+		if err = mirrorctx.epc.Close(); err != nil {
+			cx.t.Fatalf("error closing etcd processes (%v)", err)
+		}
+	}()
+
+	cmdArgs := append(cx.PrefixArgs(), "make-mirror", "--prefix", "o_", "--no-dest-prefix=true", fmt.Sprintf("localhost:%d", mirrorcfg.basePort))
```

Comment:
@gyuho i tested with your suggestion, the test case fails since its a boolean flag. What we can do instead is modify the --no-dest-prefix=true to --no-dest-prefix and it will work since flag present alone means the value = true.
Shall i modify it the way i suggested?
Thanks!